sdk/cosmos/azure-cosmos/samples/access_cosmos_with_aad.py

```diff
@@ -0,0 +1,102 @@
+import azure.cosmos.cosmos_client as cosmos_client
```

Comment:
Could we just import from `azure.cosmos`? The fact that the `cosmos_client` part of the namespace is even public is by accident... and I'd rather we didn't reinforce it in the samples :)